<launch>
    <node pkg="ros_nmpt_saliency" type="ros_nmpt_saliency_node" name="ros_nmpt_saliency_node" output="screen">
        <param name="image" value="/softkinetic/depth_registered/image" />
        <param name="point" value="/softkinetic/depth_registered/points" />
        <param name="camera_info" value="/softkinetic/depth_registered/camera_info" />
        <param name="visualize" value="true" />
        <param name="use_depth" value="true" />
    </node>
</launch>
